src/python/grpcio_tests/setup.py

```diff
@@ -59,6 +59,7 @@
     'test_lite': commands.TestLite,
     'test_gevent': commands.TestGevent,
     'test_aio': commands.TestAio,
+    'test_asyncio': commands.TestAsyncio,
```

Comment:
nit: should we use a informative name that distinguish with `test_aio`? For example, `test_sync_over_aio`, `test_lite_over_aio`, `test_wrapped_aio`?

Reply:
renamed to `test_sync_over_aio`